doc/stages/readers.ept.rst

```diff
@@ -53,7 +53,7 @@ Options
 --------------------------------------------------------------------------------
 
 filename
-    EPT resource from which to read.  Because EPT resources do not have a file extension, to specify an EPT resource as a string, it must be prefixed with ``ept://``.  For example, ``pdal translate ept://http://na.entwine.io/autzen autzen.laz``. [Required]
+    Path to the EPT resource from which to read, ending with ``ept.json``.  For example, ``/Users/connor/entwine/autzen/ept.json`` or ``http://na.entwine.io/autzen/ept.json``.  For remote paths which are not on the local filesystem, PDAL must be built with Curl support. [Required]
```

Comment:
Looking at the code, it seems that curl is now required.

Reply:
Removed Curl comment. 5abea26.